rosidl_typesupport_introspection_c/include/rosidl_typesupport_introspection_c/service_introspection.h

```diff
@@ -23,7 +23,7 @@
 
 #include "rosidl_typesupport_introspection_c/message_introspection.h"
 
-typedef struct ROSIDL_PUBLIC rosidl_typesupport_introspection_c__ServiceMembers
+typedef struct rosidl_typesupport_introspection_c__ServiceMembers
```

Comment:
The type should still be exported. See:

* https://github.com/ros2/rmw/blob/876645761751fc0be3fdaf4bf23d87b408f3e8b4/rmw/include/rmw/visibility_control.h#L39
* https://github.com/ros2/rmw/blob/876645761751fc0be3fdaf4bf23d87b408f3e8b4/rmw/include/rmw/types.h#L38